internal/db/db_blueprints.go

```diff
@@ -88,3 +102,35 @@ func (db *dB) DeleteBlueprint(id uuid.UUID, orgID, accountNumber string) error {
 	}
 	return err
 }
+
+func (db *dB) GetBlueprints(orgID, accountNumber string, limit, offset int) ([]BlueprintEntry, int, error) {
+	ctx := context.Background()
+	conn, err := db.Pool.Acquire(ctx)
+	if err != nil {
+		return nil, 0, err
+	}
+	defer conn.Release()
+
+	rows, err := conn.Query(ctx, sqlGetBlueprints, orgID, accountNumber, limit, offset)
+	if err != nil {
+		return nil, 0, err
+	}
+	defer rows.Close()
+
+	var blueprints []BlueprintEntry
+	for rows.Next() {
+		var blueprint BlueprintEntry
+		err = rows.Scan(&blueprint.Id, &blueprint.Name, &blueprint.Description, &blueprint.Version, &blueprint.CreatedAt)
+		if err != nil {
+			return nil, 0, err
+		}
+		blueprints = append(blueprints, blueprint)
+	}
+	var count int
+	err = conn.QueryRow(ctx, sqlGetBlueprintsCount, orgID, accountNumber).Scan(&count)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	return blueprints, count, nil
+}
```

Comment:
You know what, since others have the same problem let’s postpone any kind of refactoring, this is okay. Added this to: https://issues.redhat.com/browse/HMS-3308